frontend/src/components/misc/ErrorModal.tsx

```diff
@@ -32,59 +32,47 @@ class ErrorModal extends Component<ErrorModalProps> {
         const p = this.props;
 
         return <Modal
-            open={p.isVisible}
-
-            onOk={p.onClose}
-            afterClose={p.afterClose}
-
-            cancelButtonProps={{ style: { display: 'none' } }}
-
-            bodyStyle={{ paddingTop: '1.5em' }}
-            width="auto"
-            style={{
-                minWidth: 'min(95%, 550px)',
-                maxWidth: 'min(900px, 80%)',
-                width: 'auto',
-                maxHeight: '100%'
-            }}
-            centered={true}
-            maskClosable={false}
-            closeIcon={<></>}
-            transitionName={p.animate ? undefined : ''}
-            maskTransitionName={p.animate ? undefined : ''}
+            isOpen={p.isVisible}
+            onClose={p.onClose}
+            onCloseComplete={p.afterClose}
         >
-            <div style={{ display: 'flex', flexDirection: 'column', gap: '2em' }}>
-                {/* Header */}
-                <div style={{ display: 'flex', flexDirection: 'row', gap: '22px', paddingRight: '1.5em' }}>
-
-                    {/* Icon */}
-                    <div style={{
-                        height: '70px', // height determines icon size
-                        alignSelf: 'center',
-                        margin: '-6px', // compensate for built in padding
-                        marginTop: '0px',
-                    }}>
-                        <XCircleIcon color="#F53649" />
-                    </div>
-
-                    {/* Title */}
-                    <div style={{ alignSelf: 'center' }}>
-                        <div style={{
-                            color: 'hsla(0deg, 0%, 0%, 85%)',
-                            fontSize: '16px', fontWeight: 500, lineHeight: 2,
-                            overflow: 'hidden',
-                        }}>{this.title}</div>
-                        <div>{this.subTitle}</div>
-                    </div>
-                </div>
-
-                {/* Content */}
-                {this.content &&
-                    <div style={{ alignSelf: 'stretch', overflowY: 'auto', maxHeight: '300px' }}>
-                        {this.content}
-                    </div>
-                }
-            </div>
+            <ModalOverlay />
+            <ModalContent minW="3xl">
+                <ModalHeader>
+                    {this.title}
+                </ModalHeader>
+                <ModalBody>
+                    <Flex flexDirection="column" gap={8}>
+                        {/* Header */}
+                        <Flex flexDirection="row" gap={2} pr={6}>
+
+                            {/* Icon */}
+                            <div style={{
+                                height: '70px', // height determines icon size
+                                alignSelf: 'center',
+                            }}>
+                                <XCircleIcon color="#F53649"/>
+                            </div>
```

Comment:
Don't worry about this for now, but we do have an `Icon` component too, so I guess this was just not available few months ago. We can migrate it later